README.md

```diff
@@ -128,6 +128,13 @@ Schema = GraphQL::Schema.define do
 end
 </pre>
 
+When using a policy object, you may want to allow [introspection queries](http://graphql.org/learn/introspection/) to skip authorization. A simple way to avoid having to whitelist every introspection type in the RULES hash of your policy object is to check the <b>type</b> in the guard method: 
```

Comment:
could you please wrap \`RULES\`, so it's clear that we're referring a variable from the source code? :)